jsonrpc2.go

```diff
@@ -365,6 +365,11 @@ func (c *Conn) Call(ctx context.Context, method string, params, result interface
 	if err != nil {
 		return err
 	}
+	defer func() {
+		for range call.done {
+			// drain
+		}
+	}()
```

Comment:
Oops, meant to use a new goroutine here. But after discussion on Slack we decided that a buffered channel is simpler.